helpers/use-fetch.js

```diff
@@ -26,8 +26,13 @@ export default function useFetch(path, options) {
 
       try {
         const response = await fetch(url, options);
+        const data = options && options.text ? await response.text() : await response.json();
```

Comment:
you may want to move this to after the `response.status` check. that way `response.text()` or `response.json()` won't potentially error and throw before the status check